File: Java-BTL/src/Class/StudentGroup.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Class;

/**
 *
 * @author 84397
 */
public class StudentGroup {
    private String id;
    private String idCourse;
    private String name;
    private int numberOfMember;

    public StudentGroup(String id, String idCourse, String name, int numberOfMember) {
        this.id = id;
        this.idCourse = idCourse;
        this.name = name;
        this.numberOfMember = numberOfMember;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getIdCourse() {
        return idCourse;
    }

    public void setIdCourse(String idCourse) {
        this.idCourse = idCourse;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getNumberOfMember() {
        return numberOfMember;
    }

    public void setNumberOfMember(int numberOfMember) {
        this.numberOfMember = numberOfMember;
    }
    
    
}

File: New-version-2/DTO/StudentGroup.cs
using System;

public class StudentGroup
{
	private string ID;
    private string idCoruse;
    private string name;
	private int numberOfMember;
	public StudentGroup(string ID, string idCoruse, string name,int numberOfMember)
	{
		this.ID = ID;
		this.idCoruse = idCoruse; 
		this.name = name;
		this.numberOfMember = numberOfMember;
	}
	public string getID() {  return ID; }
	public string getIdCoruse() {  return idCoruse; }
	public string getName() { return name; }
	public int getNumberOfMember() {  return numberOfMember; }

}
